### ISSUE TITLE
Focus outline

### ISSUE THREAD

**Issue by ppvg** [minvws/nl-rdo-manon#421]
From independent accessibility report, SC 1.4.1:

<blockquote lang="nl">Als de niet-actieve links in het hoofdmenu focus krijgen, is dit alleen zichtbaar doordat de kleur verandert. Hierdoor is het niet voor iedereen goed te zien dat deze elementen focus krijgen. Het advies is om de focus aan te geven door middel van een omlijning in plaats van enkel een kleurverandering.</blockquote>

The `soft` theme is missing a focus outline on various elements, including the main nav link.

**Comment by StephanoMarshall** [comment]
Blocked by tech debt (tickets to be created)
